config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -1001,7 +1001,7 @@ presubmits:
         - --cluster=
         - --extract=local
         - --gcp-node-image=gci
-        - --gcp-zone=us-west1-b
+        - --gcp-zone=us-central1-f
```

Comment:
west1-b ?

Reply:
ack

Reply:
done (though it doesn't really matter anymore)